spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskExecutionService.java

```diff
@@ -853,26 +853,6 @@ private boolean isAppDeploymentSame(TaskManifest previousManifest, TaskManifest
 	 */
 	@Override
 	public String getLog(String platformName, String taskId, String schemaTarget) {
-		Launcher launcher = this.launcherRepository.findByName(platformName);
```

Comment:
I ❤️ all these deletes! 

This does make me realize that we should really firm up the javadocs on `DefaultTaskExecutionService.getLog` to clearly spell out what the taskId is expected to be. Before this change, it was `taskName` but the param is called `taskId`. 
```java
	@Override
	public String getLog(String platformName, String taskId, String schemaTarget) {
```